src/components/back-to-top/btt.dom.js

```diff
@@ -0,0 +1,12 @@
+import domready from '../../js/domready';
+
+async function btt() {
```

Comment:
Think we should have a more descriptive name here rather than btt

Reply:
Table of Content uses toc, do we need to change that too?

Reply:
Maybe, just when initially looking at your code I saw "btt" and thought it could be more obvious what that function was for

Reply:
sure, ill change it for back to top right now, and we can ask around in a tech session for change of ToC